api/query/atoms.go

```diff
@@ -146,6 +149,22 @@ func (c AbstractCount) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
 	}
 }
 
+// AbstractLink is represents the root of a link query, whic matches Metadata URLs
+// to a pattern string; it is independent of test runs.
+type AbstractLink struct {
+	Pattern string
+}
+
+// BindToRuns for AbstractLink is a no-op; it is independent of test runs
+func (l AbstractLink) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
+	var netClient = &http.Client{
+		Timeout: time.Second * 5,
+	}
+	metadata := shared.GetMetadataResponse(runs, netClient, log.StandardLogger())
```

Comment:
We only do query binding in a request, right? Is it possible to get a logger from the request context instead of using the standard logger?